static/js/common.js

```diff
@@ -131,3 +148,11 @@ export function adjust_mac_shortcuts(key_elem_class, require_cmd_style) {
         $(this).text(key_text);
     });
 }
+
+// See https://zulip.readthedocs.io/en/latest/development/authentication.html#password-form-implementation
+// for details on implementation.
+export function setup_password_visibility_toggle(password_field_id, password_selector) {
+    $("body").on("click", password_selector, () => {
+        toggle_password_visibility(password_field_id, password_selector);
```

Comment:
Unless there's a good reason not to, click handlers that handle an event should use `e.preventDefault` and `e.stopPropagation` -- not doing so can result in unexpected behavior where a click does multiple things in a confusing way.

Reply:
I pushed back with this change, since I'd made it manually before deciding this wasn't close enough for me to finish and merge today.  Also made a very small tweak to the `docs/` (changed 1-2 words) for better English readability.

Reply:
Okay.

Reply:
This is giving the error `Cannot read property preventDefault() of undefined` so the parameter passed in the click handler is undefined.

Reply:
`e` needs to be defined @ligmitz 
Like see you can do this way

![image](https://user-images.githubusercontent.com/56730716/112747621-274b0a00-8fd4-11eb-8f76-d24104f58173.png)